rest/model/string.go

```diff
@@ -1,22 +1,14 @@
 package model
 
-import (
-	"encoding/json"
-)
+type APIString *string
 
-type APIString string
-
-func (as *APIString) MarshalJSON() ([]byte, error) {
-	if *as == "" {
-		return []byte("null"), nil
-	}
-	str := string(*as)
-	return json.Marshal(&str)
+func ToApiString(in string) APIString {
+	return APIString(&in)
```

Comment:
For `ToApiString` and `FromApiString` to be reciprocal, this function would need to return a nil pointer in the case where `in == ""`. Can you explain why you chose this implementation instead?

Reply:
I don't intend them to be inverses of each other, since one of the data types in the conversion is more expressive than the other. Returning a blank json string for a blank db string feels like the correct behavior